backend/company-agent/src/main/java/org/hyperledger/oa/impl/aries/AriesStartupTasks.java

```diff
@@ -55,9 +54,15 @@
     @Inject
     Optional<SchemaService> schemaService;
 
+    @Inject
+    EndpointService endpointService;
+
     @Inject
     Optional<PartnerCredDefLookup> credLookup;
 
+    @Value("${oagent.web.only}")
+    private Boolean webOnly;
```

Comment:
If you are in the Aries Startup Task you will never be in the web mode, this has already been taken care of. See @RequiresAries & @RequiresWeb annotations on the delegated StartupTasks. Please remove this.

Reply:
good point